inspec.lock

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Can we remove this `inspec.lock` file pls?

Reply:
Done - sorry!